js/two-factor.js

```diff
@@ -0,0 +1,35 @@
+/* global ajaxurl */
+jQuery(document).ready(function ($) {
+
+	$('.delete').on('click', function( e ) {
```

Comment:
This is to generic. We need to give additional specificity because the likelihood of another `.delete` existing is very high.